examples/drone_usage_example.rs

```diff
@@ -0,0 +1,66 @@
+//THIS IS JUST AN EXAMPLE OF IMPLEMENTATION
+
+use crate::api::drone::{Drone, DroneImplement};
+use crate::types::packet::{Packet, PacketType};
+use crossbeam_channel::{select, Receiver, Sender};
+use std::any::Any;
+use std::collections::HashMap;
+use std::thread;
+
+fn main() {
+    let handler = thread::spawn(move || {
+        //Create some drone
+        // let drone = Drone::new(...);
+
+        //Then start the drone
+        //(not forced to use '.run()', a loop is enough).
+        // drone.run();
```

Comment:
run should be part of DroneImplement so that simulation controller can threat them as opaque. Without needing to knoe how to start each single drone.

Reply:
You're right, I'll move it.